validator/gcp_iam_restrict_service_account_creation.rego

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPIAMRestrictServiceAccountCreationConstraintV1
+
+import data.validator.gcp.lib as lib
+
+deny[{
+	"msg": message,
+	"details": metadata,
+}] {
+	constraint := input.constraint
+	asset := input.asset
+	asset.asset_type == "google.iam.ServiceAccount"
+	service_account := asset.resource.data
+	service_account_email := service_account.email
+
+	# TODO(johngao): should include project id.
+endwith(	service_account_email, "iam.gserviceaccount.com")
```

Comment:
this format looks a bit odd?

Reply:
Yes, this line looks misformatted. Also can you fix the TODO (or remove it)?

Reply:
Done.